README.md

```diff
@@ -1,95 +1,117 @@
 
-Hardware specification:
-http://www.mindpx.net/assets/accessories/Specification_MindPX.pdf
+#### Open Reference Hardware
 
-User manual:
-http://www.mindpx.net/assets/accessories/UserGuide_MindPX.pdf
+The project provides *open reference hardware/layouts* for **some** open designs, in some versions.
+These are provided in the form of PCB layout files.
 
-Key features of v2.8:
+The reference hardware files are shared under the same [license](#licenses) as the associated open design, and hence may be used in the same way(s).
 
-1. 8 main PWM outputs + 8 aux PWM outputs
-2. Dual-redundancy IMU components (onboard HMC5883L compass)
-3. Support PPM/SBUS/SPEKTRUM DSM/Satellite RC input
-4. Support FrSky/Wifi telemetry
-5. One extra I2C port (so can support external compass and px4flow simultaneously)
-6. Secondary USB port  for companion computer (converted from UART2 using onboard CP2104)
-7. nrf/SPI port for SPI devices
-8. exposed SWD debug port
-9. Built-in vibration dampening for isolated IMU, support hard mount on air frames.
-10. LTC2875 as CAN controller (+/- 60V overvoltage fault protection)
+#### FMU Versions
 
-MindRacer V1.2 board (Hardware version: v1.2)
+The Pixhawk project has evolved the FMU design through a number of verisons.
 
-MindRacer specification:
-http://mindpx.net/assets/accessories/mindracer_spec_v1.2.pdf
+These are named using the designation: FMUvX (e.g.: FMUv1, FMUv2, FMUv3, FMUv4, etc.).
+Higher FMU numbers indicate that the board is more recent, but may not indicate increased capability (versions can be almost identical - differing only in connector wiring).
 
-MindRacer user manual:
-http://mindpx.net/assets/accessories/mindracer_user_guide_v1.2.pdf
+The versions are listed below (with a high level overview of the main differences).
 
-Key features of v1.2:
+Version | Year | Lead Product | CPU | Other
+--- | --- | --- | --- | ---
+[FMUv1](FMUv1/README.md) & [IOv1](IOv1) | 2012 | None | 168 MHz M4 | (Discontinued) Original Flight Management Unit and Separate I/O board.
+[FMUv2](FMUv2/README.md) | 2013 | Pixhawk 1 | 168 MHz M4 | Single board with STM32427VI processor.
+[FMUv3](FMUv3_REV_D/README.md) | 2015 | Pixhawk 2 | 168 MHz M4 | Redundant sensors. Identical to FMUv2, but usable flash doubled to 2MB.
+[FMUv4](FMUv4/README.md) | 2015 | Pixracer | 168 MHz M4 | Increased RAM. Faster CPU. More serial ports. No IO processor.
+FMUv4X | 2017 | Pixhawk 3 Pro | 168 MHz M4 | Slightly increased RAM. More serial ports. IO processor. Redundant sensors.
+[FMUv5](FMUv5/README.md) | 2018 | 200 MHz M7 | Pixhawk 4 | New processor (F7). Much faster. More RAM. More CAN busses. Much more configurable.<br>> **Note** Minimum specification provided (pinout info, but no schematics).
+FMUv5X | 2019 | Pixhawk 5X | 200 MHz M7 | Temp-calibrated, redundant sensors.
```

Comment:
1. OK to add a folder for FMUv5X and add a README linking to the [DRAFT] Pixhawk Reference Standard ?
1. OK to add the "future" products here?